test/addressSpec.js

```diff
@@ -0,0 +1,146 @@
+var addr = require('../src/js/address');
+
+describe('address functions', function() {
+  describe('is adddress duplicate', function() {
+    var address,
+      duplicates;
+
+    beforeEach(function(){
+      address = '123 Main St., Somecity, SomeWhere, 12345';
```

Comment:
huh, it sure does. Ignore this ^^; Plus that's probably a premature optimization anyway.. having it this way is less likely to break imo, especially since with just one variable you start relying on state between tests which is a no go, imo (could cause cascading failures and make bugs harder to track)